modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -273,20 +478,82 @@ public void Add(Variant item)
         }
 
         /// <summary>
-        /// Checks if this <see cref="Array"/> contains the given item.
+        /// Adds the elements of the specified collection to the end of this <see cref="Array"/>.
+        /// </summary>
+        /// <param name="collection">Collection of <see cref="Variant"/> items to add.</param>
+        /// <exception cref="ArgumentNullException">
+        /// The <paramref name="collection"/> is <see langword="null"/>.
+        /// </exception>
+        public void AddRange(IEnumerable<Variant> collection)
```

Comment:
This should work for `Variant[]` and `List<Variant>` but not `int[]` or `List<int>` (even though `int` is implicitly convertible to `Variant`). We may want to implement it generically instead:

```csharp
public void AddRange<T>(IEnumerable<[MustBeVariant] T> collection);
```

Reply:
The generic version would be fine.